<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuspensionesyActivaciones</class>
 <widget class="QMainWindow" name="SuspensionesyActivaciones">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tester Contenido</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>471</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Listado Clientes CSV</string>
    </property>
    <widget class="QPushButton" name="btn_ingrezar_csv">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar Archivo</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>471</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Clientes en Mikrotiks </string>
    </property>
    <widget class="QPushButton" name="btn_clientes">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Obtener Clientes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_clientes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sin Cargar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>951</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Suspender o Activar</string>
    </property>
    <widget class="QPushButton" name="btn_suspender">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Suspender Clientes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_activar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>70</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Activar Clientes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_proceso">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Proceso en Ejecuci√≥n</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
